.github/workflows/release.pub

```diff
@@ -0,0 +1,2 @@
+untrusted comment: minisign public key: DE23EFE715137751
```

Comment:
What is this from/for?

Reply:
A signing key for signing checksums - I use it on pulumi-watch. I'll remove it, as we need to discuss this as a team. It's unused (the sign job is disabled.)